ocs_ci/utility/utils.py

```diff
@@ -832,17 +836,22 @@ def get_openshift_installer(
         log.debug(f"Installer exists ({installer_binary_path}), skipping download.")
         # TODO: check installer version
     else:
-        version = expose_ocp_version(version)
+        if config.ENV_DATA.get("fips"):
+            # WA issue https://github.com/red-hat-storage/ocs-ci/issues/9755
+            version = "4.16.0-0.nightly-2024-04-26-145258"
```

Comment:
should this be done only on OCP 4.16? What about upgrade scenario where we install N-1?

Reply:
yes, its only for 4.16. didn't test the upgrade scenario and I hope upgrade will be fine.

Reply:
done